plugins/newsdownloader.koplugin/main.lua

```diff
@@ -17,9 +17,22 @@ local NewsDownloader = WidgetContainer:new{}
 local initialized = false  -- for only once lazy initialization
 local FEED_CONFIG_FILE = "feeds.xml"
 local FILE_EXTENSION = ".html"
-local FEED_SOURCE_SUFFIX = "_rss_tmp.xml"
+local NEWS_DL_DIR_NAME = "news"
```

Comment:
you don't have to hack apk file with the new custom plugin look path feature. you can now drop plugins in your data directory so they won't get wiped with apk update :)